net-analyzer/zeek/metadata.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+<!-- maintainer-needed -->
```

Comment:
A package cannot be added without a maintainer ;) You should probably list yourself along with proxy-maint.

Reply:
Done

Reply:
Done here and for the dependency lib